connectors/sources/tests/test_generic_database.py

```diff
@@ -338,7 +342,9 @@ async def test_fetch_documents_negative(patch_logger):
         "execute_query",
         side_effect=InternalClientError("Something went wrong"),
     ):
-        source.engine = create_engine("oracle://admin:changme@127.0.0.1:1521/testdb")
+        source.engine = create_engine(
+            "oracle+oracledb://admin:changme@127.0.0.1:1521/testdb"
```

Comment:
Also seems to be a constant used in multiple places. You could create a `CONNECTION_STRING` constant and reuse it.